pyogrio/tests/test_geopandas_io.py

```diff
@@ -17,19 +17,19 @@
     import geopandas as gp
     from geopandas.testing import assert_geodataframe_equal
 
-    has_geopandas = True
+    from shapely.geometry import Point
 except ImportError:
-    has_geopandas = False
+    pass
+
+
+pytest.importorskip("geopandas")
```

Comment:
Using `importorskip` instead of `pytestmark = pytest.mark.skipif` makes that we can use those imported functions in parametrization setup (otherwise that still errors at collection time, since the pytestmark doesn't avoid collecting all tests in the file)